src/components/include/application_manager/hmi_capabilities.h

```diff
@@ -433,28 +436,11 @@ class HMICapabilities {
   virtual void set_navigation_capability(
       const smart_objects::SmartObject& navigation_capability) = 0;
 
-  /*
```

Comment:
why remove these function descriptions?

Reply:
@LuxoftAKutsan Descriptions haven't been added back in?